### PR TITLE
Doxygen in CALVOR.f, CALWXT_BOURG.f, CLDRAD.f, COLLECT.f, COLLECT_LOC.f, DEALLOCATE.f, and DEALLOCATE.f

### PR DESCRIPTION
**Pull request by kayeekayee**
This is part of Issue #392. 

Fixes the doxygen warnings in CALVOR.f, CALWXT_BOURG.f, CLDRAD.f, COLLECT.f, COLLECT_LOC.f, DEALLOCATE.f, and DEALLOCATE.f.

Please review the changes. Thank you.

### CODE REVIEW

File: sorc/ncep_post.fd/CLDRAD.f [hunk 1]
```diff
@@ -107,19 +79,18 @@ SUBROUTINE CLDRAD
                          HBOT, HBOTD, HBOTS, HTOP, HTOPD, HTOPS,  FIS, PBLH,  &
                          PBOT, PBOTL, PBOTM, PBOTH, CNVCFR, PTOP, PTOPL,      &
                          PTOPM, PTOPH, TTOPL, TTOPM, TTOPH, PBLCFR, CLDWORK,  &
-                         ASWIN, AUVBIN, AUVBINC, ASWOUT,ALWOUT, ASWTOA,       &
```
Comment (WenMeng-NOAA) [inline sorc/ncep_post.fd/CLDRAD.f]: @kayeekayee It seems you invert the changes in PR #453. Please remove duplicated ASWIN.

File: sorc/ncep_post.fd/CLDRAD.f [hunk 2]
```diff
@@ -107,19 +79,18 @@ SUBROUTINE CLDRAD
...
+                         ASWIN, AUVBIN, AUVBINC, ASWIN, ASWOUT,ALWOUT, ASWTOA,&
                          RLWTOA, CZMEAN, CZEN, RSWIN, ALWIN, ALWTOA, RLWIN,   &
                          SIGT4, RSWOUT, RADOT, RSWINC, ASWINC, ASWOUTC,       &
                          ASWTOAC, ALWOUTC, ASWTOAC, AVISBEAMSWIN,             &
-                         AVISDIFFSWIN, ASWINTOA, ASWTOAC, AIRBEAMSWIN,        &
+                         AVISDIFFSWIN, ASWINTOA, ASWINC, ASWTOAC, AIRBEAMSWIN,&
```
Comment (WenMeng-NOAA) [inline sorc/ncep_post.fd/CLDRAD.f]: @kayeekayee The same reason as above. Please revert your change.

File: sorc/ncep_post.fd/CLDRAD.f [hunk 3]
```diff
@@ -107,19 +79,18 @@ SUBROUTINE CLDRAD
...
                          AIRDIFFSWIN, DUSMASS, DUSMASS25, DUCMASS, DUCMASS25, &
                          ALWINC, ALWTOAC, SWDDNI, SWDDIF, SWDNBC, SWDDNIC,    &
                          SWDDIFC, SWUPBC, LWDNBC, LWUPBC, SWUPT,              &
                          TAOD5502D, AERSSA2D, AERASY2D, MEAN_FRP, LWP, IWP,   &
                          AVGCPRATE,                                           &
                          DUSTCB,SSCB,BCCB,OCCB,SULFCB,DUSTPM,SSPM,aod550,     &
-                         du_aod550,ss_aod550,su_aod550,oc_aod550,bc_aod550,   &
-                         PWAT
```
Comment (WenMeng-NOAA) [inline sorc/ncep_post.fd/CLDRAD.f]: @kayeekayee The same reason as above. Please revert your change.

File: sorc/ncep_post.fd/CLDRAD.f [hunk 4]
```diff
@@ -419,29 +389,12 @@ SUBROUTINE CLDRAD
       IF (IGET(080) > 0) THEN
 ! dong 
          GRID1 = spval
-         Model_Pwat = .false.
```
Comment (WenMeng-NOAA) [inline sorc/ncep_post.fd/CLDRAD.f]: @kayeekayee @kayeekayee The same reason as above. Please revert your change.

**Comment by WenMeng-NOAA** [comment]
@kayeekayee Please sync your branch up to date for merging process.

**Comment by kayeekayee** [comment]
Hi Wen, do you know why my code inverted your changes? I had my code fetched upstream before I submitted my PRs.

**Comment by WenMeng-NOAA** [comment]
> Hi Wen, do you know why my code inverted your changes? I had my code fetched upstream before I submitted my PRs.

@kayeekayee I am not sure why some of upstream changes were not synced in your branch. Usually I always start my PR 
 feature branch from upstream/develop instead of develop in my fork and never reuse the feature branch which was merged into upstream/develop. Sometime, there are conflicts during syncing with upstream/develop. You might  solve the conflicts one by one.

**Comment by kayeekayee** [comment]
> 

Ok I will be aware of this issue. Thank you.

**Comment by kayeekayee** [comment]
Sorry I closed this PR by mistake!!!

**Review by WenMeng-NOAA — APPROVED** [review]

**Comment by WenMeng-NOAA** [comment]
The UPP RT tests were done on Hera. No baseline updates are needed.

**Pull request merged**